shopelectro/models.py

```diff
@@ -82,6 +82,8 @@ class Product(
         verbose_name=_('tags'),
     )
 
+    # @todo #910:30m  Explore and doc what is vendor_code.
+    #  And render it as some hidden option on the product page.
```

Comment:
>render it as some hidden option

What did you mean? Should we remove this from product page?

Reply:
i'll just rm this string